client/src/components/Forms.js

```diff
@@ -92,7 +92,7 @@ const StartForm = (props) => {
             <Label htmlFor="mpw">How many Miles Per Week (MPW) do you currently
                   run?</Label>
             <Input  onChange={props.onChange} type="select" name="mpw">
-              <option value="0">0</option>
+              <option value="0">1</option>
```

Comment:
You want to set the value, not the inner text. Value is what is sent when submitted, inner text is what shows up on the site.